### DISCUSSION TITLE
Nested groups support

### DISCUSSION THREAD

**Discussion by Que-tin** [petyosi/react-virtuoso#1047] [Ideas]
It would be great if there would be nested group support. I have a case were I need to add an additional nesting layer too an existing `GroupedVirtuoso`. I think as of now I have no option but to either get rid of virtuoso or accept, that my subgroup is not virtualized.

Are there any plans on that or is it already possible and I'm missing smth.? The type hint that it's not possible as `groupCounts` is a number array and does not accept any type of deeper nesting.

**Comment by petyosi** [comment]
Are you willing to sponsor the development of such feature? I'm also open to a PR, if you feel like it.
